### ISSUE TITLE
Support weight/bias-wise learning rate setting

### ISSUE THREAD

**Issue by wkentaro** [wkentaro/pytorch-fcn#1]
As in https://github.com/shelhamer/fcn.berkeleyvision.org/blob/master/voc-fcn32s/net.py#L8

**Comment by wkentaro** [comment]
https://github.com/wkentaro/pytorch-fcn/blob/master/examples/voc/train_fcn32s.py#L105
